tcms/testplans/templates/testplans/mutable.html

```diff
@@ -0,0 +1,100 @@
+                    <input type="text" id="id_name" name="name" value="{{ form.name.value }}" class="form-control">
+                </div>
+            </div>
+
+            <div class="form-group">
+                <div class="col-md-1 col-lg-1">
+                    <label for="id_product">{% trans "Product" %}</label>
+                    <a href="{% url 'admin:management_product_add' %}?_popup" id="add_id_product" alt="{% trans 'add new Product' %}" title="{% trans 'add new Product' %}">+</a>
+                </div>
+                <div class="col-md-3 col-lg-3">
+                    <select id="id_product" name="product" class="form-control">
+                        {% for product in form.product.field.queryset %}
+                            <option value="{{ product.pk }}">
+                                {{ product.name }}
+                            </option>
+                        {% endfor %}
+                    </select>
+                </div>
+                <div class="col-md-1 col-lg-1">
+                    <label for="id_version">{% trans "Version" %}</label>
+                    <a href="{% url 'admin:management_version_add' %}?_popup"
+                       id="add_id_version" alt="{% trans 'add new Version' %}" title="{% trans 'add new Version' %}">
+                        +
+                    </a>
+                </div>
+                <div class="col-md-3 col-lg-3">
+                    <select id="id_version" name="version" class="form-control">
+                        {% for product_version in form.product_version.field.queryset %}
+                            <option value="{{ product_version.pk }}">
+                                {{ product_version.value }}
+                            </option>
+                        {% endfor %}
+                    </select>
+                </div>
+                <div class="col-md-1 col-lg-1">
+                    <label for="id_type">{% trans "Type" %}</label>
+                </div>
+                <div class="col-md-3 col-lg-3">
+                    <select class="form-control" id="id_type" name="type">
+                        {% for type in form.type.field.queryset %}
+                            <option value="{{ type.pk }}"> {{ type }} </option>
+                        {% endfor %}
+                    </select>
+                </div>
+            </div>
+
+            <div class="form-group">
+                <label class="col-md-1 col-lg-1" for="id_parent_id">{% trans "Parent ID" %}</label>
+                <div class="col-md-3 col-lg-3">
+                    <input type="text" id="id_parent_id" name="parent_id" value="" class="form-control">
+                </div>
+                <label class="col-md-1 col-lg-1" for="id_product">{% trans "Reference link" %}</label>
+                <div class="col-md-7 col-lg-7">
+                    <input type="text" id="id_summary" name="summary" value="" class="form-control">
```

Comment:
id and name mismatch ? Doesn't this produce an error when you try to save the TestPlan ?

Reply:
No. The `id` is for js/css related stuff, while `name` is for the form. It seems to be fine

Reply:
I see you've updated it with the correct values :)